Add PR and issue templates. (#73)
PyPSA/atlite: .github/pull_request_template.md

Closes # (if applicable).

## Change proposed in this Pull Request

<!--- Provide a general, short summary of your changes in the title above -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Type of change
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I tested my contribution locally and it seems to work fine.

> - [ ] I locally ran `pytest` inside the repository and now unexpected problems came up.

Reviewer: ```suggestion
- [ ] I locally ran `pytest` inside the repository and no unexpected problems came up.
```

> - [ ] I have documented the changes to the code.

Reviewer: it's not very clear what is meant here, perhaps better
```suggestion
- [ ] I have adjusted the docstrings in the code appropriately.
```
